config/routes.rb

```diff
@@ -3,5 +3,9 @@
 
   devise_for :users
 
-  resources :events, only: %i[new create update destroy]
+  resources :events, only: %i[create update destroy] do
+    resources :bookings, only: %i[create] do
+      patch :cancel_booking, on: :member
+    end
+  end
```

Comment:
Can you change the name of the route to cancel and give it an alias of :cancel_booking ?

Reply:
```
  resources :events, only: %i[create update destroy] do
    resources :bookings, only: %i[create] do
      patch :cancel, as: :cancel_booking, on: :member
    end
  end
 ```